modules/install-openshift-common-terms.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * installing/index.adoc
+
+:_content-type: REFERENCE
+[id="install-openshift-common-terms_{context}"]
+= Common terms
+
+This glossary defines common terms that are used in the installation content. These terms help you understand installation effectively.
+
+Control plane::
+Th control plane manages the worker nodes and the pods in the {product-title} cluster. It is a container orchestration layer that exposes the API and interfaces to define,deploy, and manage the lifecycle of containers.
```

Comment:
That doesn't sound very accurate for me. In IPI, it should be machine-api that manages the worker nodes, in UPI, the user manages the worker nodes.

Reply:
I found there's already some comprehensive description on `Control planes` in https://docs.openshift.com/container-platform/4.10/architecture/control-plane.html#defining-masters_control-plane